test/python/requirements.txt

```diff
@@ -1,2 +1,3 @@
 lit
 pytest
+torch==2.3.0 --index-url https://download.pytorch.org/whl/cpu
```

Comment:
this is needed because `test/python` rely on torch which is installed through ttrt, but if ttrt installation doesn't happen before this test is executed it leads to test failures like here:
https://github.com/tenstorrent/tt-mlir/actions/runs/11329283307/job/31528275878